build.sbt

```diff
@@ -150,11 +150,31 @@ lazy val otelExtension = (project in file("otel-extension"))
   )
   .dependsOn(core % "provided->compile;compile->compile", testkit % "it,test")
 
-lazy val example = (project in file("example"))
-  .enablePlugins(JavaAppPackaging, UniversalPlugin)
+def exampleCommonSettings = Seq(
+  publish / skip := true,
+  assemblyMergeStrategySettings,
+  resolvers += "Sonatype OSS Snapshots" at "https://oss.sonatype.org/content/repositories/snapshots",
+  run / javaOptions ++= {
+    val properties = System.getProperties
+
+    import scala.collection.JavaConverters._
+    val keys = for {
+      (key, value) <- properties.asScala.toList if value.nonEmpty
+    } yield s"-D$key=$value"
+
+    keys
+  },
+  run / javaOptions ++= Seq(
+    s"-javaagent:$projectRootDir/opentelemetry-javaagent-$OpentelemetryVersion.jar",
+    s"-Dotel.javaagent.extensions=${(otelExtension / assembly).value.absolutePath}",
+    "-Dotel.javaagent.debug=true"
+  )
+)
+
+lazy val exampleAkka = (project in file("examples/akka"))
+  .settings(exampleCommonSettings)
   .settings(
-    name           := "mesmer-akka-example",
-    publish / skip := true,
+    name := "mesmer-akka-example",
     libraryDependencies ++= {
```

Comment:
What would you say about cleaning this up and moving the dependencies to Dependencies.scala like the rest of the deps?

Reply:
eg. have zioExampleDependencies / akkaStreamingExampleDependencies etc in Dependencies.scala and reduce duplication in that file?

Reply:
I do not see what is really solved by having `zioExampleDependencies`, `akkaStreamingExampleDependencies`, so I decided not to go with them, but I think I removed the last of duplication in example project settings. I am open to changing my mind though. 

Maybe moving all the library definitions to the `Dependencies.scala` looks nice, but maybe for example project it does not matter as much, especially if those dependencies are only used for examples. I guess we just have to agree on something and stick to it 😄

Reply:
Might be personal preference but I feel like it's nicer to have all the deps in a separate file that is dedicated to deps only. That being said, if you're not convinced I'm ok with leaving as is because it's not very important to me.